qemu/tests/cfg/enable_scatter_windows.cfg

```diff
@@ -19,7 +19,7 @@
     autoit_name = "AutoIt3_%PROCESSOR_ARCHITECTURE%.exe"
     install_winpcap_cmd = "WIN_UTILS:\${autoit_name} WIN_UTILS:\install_winpcap.au3"
 
-    param_names = "Offload.TxLSO Offload.TxLSO MTU"
+    param_names = "Offload.TxLSO Offload.TxLSO *JumboPacket"
```

Comment:
In order to maintain downward compatibility, please introduce a judgment to determine whether to use MTU or JumboPacket.